src/client/styles/checkbox-field-input.css

```diff
@@ -0,0 +1,3 @@
+.chakra-checkbox__input {
+    scroll-margin-top: 88px;
```

Comment:
Is it possible to override this value [using Chakra](https://chakra-ui.com/docs/theming/component-style#styling-single-part-components) instead of patching the CSS directly? How Chakra name their CSS classes should be transparent to us.

Reply:
It might be possible to override the value through overriding the global default input themes, but I feel that this solution might be too invasive and cause unexpected behaviours on modals where we may want to customise the same set of styles on another page with different values.

As such, I'd still prefer that we just override the CSS directly, though we could discuss about this if you have strong opinions on the subject.

Reply:
As discussed on Slack, this current way of overriding the CSS also affects _all_ checkbox inputs since it's overriding global styles.